캐스퍼 추가

yoonch9009 opened the pull request:
fingerprints 추출하여 캐스퍼 추가하였으나 CAN Error가 발생합니다.

tmux a 로 봐도 특별한 로그가 뜨는게 아니라 원인을 찾기 어렵습니다.

혹시 해결방법 제안해 주실수 있을까요..?

![20240616_205551](https://github.com/ajouatom/openpilot/assets/34851946/fd0233d4-8228-4f1e-82b8-2b2b46b80b82)

Commit:
openpilot v0.9.4 release

date: 2023-07-27T18:38:32
master commit: fa310d9e2542cf497d92f007baec8fd751ffa99c

Commit:
openpilot v0.9.5 release

date: 2023-11-17T23:53:40
master commit: d3aad9ca4601ae0a448ed971c1cd151c7c1eb690

Commit:
openpilot v0.9.6 release

date: 2024-02-21T23:02:42
master commit: 0b4d08fab8e35a264bc7383e878538f8083c33e5

Commit:
openpilot v0.9.7 release

date: 2024-06-11T01:36:39
master commit: f8cb04e4a8b032b72a909f68b808a50936184bee

ajouatom commented:
혹시 레이더 배선개조했나요?
배선개조했다면 SCC connected BUS2를 1로,

배선개조가 아니라면 CAMERA SCC인지도 모르겠네요..

그래도 SCC connected BUS2를 1로,

결과를 알려주시면 정식으로 등록할께요..

yoonch9009 commented:
> 혹시 레이더 배선개조했나요?
> 배선개조했다면 SCC connected BUS2를 1로,
> 
> 배선개조가 아니라면 CAMERA SCC인지도 모르겠네요..
> 
> 그래도 SCC connected BUS2를 1로,
> 
> 결과를 알려주시면 정식으로 등록할께요..
> 

레이더 배선개조는 아직 하지 않았습니다.
커넥터랑 선 구매는 다 했지만 먼저 오픈파일럿만 연결해 보고자 했었는데, 그래도 안되면 배선개조까지 해보고 SCC connected BUS2도 변경해 봐야겠네요.

추후 결과 알려드리겠습니다. 감사합니다.

-----
SCC connected BUS2를 1로 바꾸니 인식은 되고, 조향값도 잘 받아오는것 같은데 실제 핸들이 움직이지 않습니다.
이번엔 에러도 전혀 안떠서 왜그런지 모르겠네요..

ajouatom commented:
캐스퍼 돌아가요?

yoonch9009 commented:
핸들 제어가 전혀 안됩니다..
이것저것 수정해보고 다른 branch도 여러개 돌려봤는데도 도저히 제어가 안되네요 ㅜㅜ
방법이 있을까요?

![1](https://github.com/user-attachments/assets/6ef7942c-121a-4ecf-b3ad-c7dc1a4fa9ed)
![2](https://github.com/user-attachments/assets/013cc896-4d5b-4d37-99a8-8498c8228e65)

ajouatom committed:
carrot2-v6-wip4

Commit:
openpilot v0.9.8 release

date: 2024-12-07T09:09:02
master commit: 7a4169379df03040e3958e5626d6097503f2d542

Kirito3481 committed:
Change select car ui (#108)

ajouatom committed:
fix hda2 4b4 useless code....

ajouatom committed:
Update input.cc
